pyvortex/src/expr.rs

```diff
@@ -247,15 +262,12 @@ impl PyExpr {
 pub fn column<'py>(name: &Bound<'py, PyString>) -> PyResult<Bound<'py, PyExpr>> {
```

Comment:
I don't think we're really committed to API compatibility. Seems like we should rename this to `get_item` now.

Reply:
Erm. I guess that's a bit weird for pythonistas because of the builtin, but I still don't want to have a difference in naming between Python and Rust.

Reply:
I like the consistency with other python data-ish libraries/ecosystem, makes it more familiar. We also have `__get_item__` which works over other existing expressions (like identity).